CONTRIBUTING.md

```diff
@@ -56,6 +56,7 @@ def count_truthy(elements: List[Any]) -> int:
 
 - Keep UI elements fully keyboard accessible, and aim to support screen-readers where possible
 - Maintain a high level of animation performance, and minimise page-load time
+- We use a specific set of colours which is defined in the style guide docs under theme colours.   
```

Comment:
I would  reword this and also include the document to be more in line with the style of our docs:
_We use the set of colours defined in the ![Kedro-Viz Color Styleguide](./.github/img/kedro_colours.png) for all colour variables and usage. _